Packs/Anomali_ThreatStream/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Anomali ThreatStream",
     "description": "Use Anomali ThreatStream to query and submit threats.",
     "support": "xsoar",
-    "currentVersion": "2.1.12",
+    "currentVersion": "2.2.0",
```

Comment:
This is a very minor change, should be a revision version bump (2.1.12 --> 2.1.3).